Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/InputActionsEditorView.cs

```diff
@@ -65,6 +78,12 @@ public void BuildUI()
                 });
         }
 
+        private void OnReset()
+        {
+            var asset = ProjectWideActionsAsset.ResetAssetToDefault();
```

Comment:
I don't think this belongs in the View.
There is nothing else like it here (anything dealing with state).
I think you need to `Dispatch` a new Command from here to keep it consistent with the reset of the code.